keps/sig-node/2535-ensure-secret-pulled-images/README.md

```diff
@@ -350,8 +696,11 @@ Yes.
 
 ###### What happens if we reenable the feature if it was previously rolled back?
 
-Images pulled during the period the feature was disabled will not be present in the cache, and thus could incur redundant pulls/container creation failures.
-However, the cache may still be present, and thus it will retain information from when it was previously enabled.
+Images pulled during the period the feature was disabled will not be present in the cache. Depending on the policy configured,
+this could either incur redundant pulls/container creation failures OR would cause the feature to not work properly for
+images pulled during the time the feature was disabled.
```

Comment:
When the policy is "AlwaysVerify" and registry is unavailable then the containers will not be able to create.  Clarify this more precisely?

"cause the feature to not work properly" - Clarify that the images pulled during this time will be treated.

Reply:
Good suggestion, see if the latest change addresses your comment :+1: